### PR TITLE
Bug 2067775: Update prometheus client_golang from 1.11.0 => 1.11.1

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: go.mod
Comment (gabemontero) [inline go.mod:37]: same as your other PR .... either put this with the other requires, or put a comment explaining why it is separated

Comment (coreydaley) [reply go.mod]: updated.

File: go.mod
Comment (adambkaplan) [inline go.mod:196]: Do we know where we are pulling this as an indirect dependency? For instance, if we are getting it from k8s, we should see if there is a bugfix to the k8s libraries.

Comment (coreydaley) [reply go.mod:196]: I looked at the k8s libraries and the latest version was not updated yet.

Comment (adambkaplan) [reply go.mod:196]: Running `go mod graph | grep "prometheus/client_golang"`, lots of things depend on client_golang.
One of them is openshift/library-go - we could do the following:

1. Update library-go with the dependency update.
2. Re-vendor library-go here.

https://github.com/openshift/library-go/blob/master/go.mod#L24

Comment (coreydaley) [reply go.mod:196]: Opened https://github.com/openshift/library-go/pull/1349 and will update this pull request once that merges

**Comment by coreydaley** [comment]
/hold
Waiting on https://github.com/openshift/library-go/pull/1349

**Comment by coreydaley** [comment]
@adambkaplan Updated library-go , ptal

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Pull request merged**
